README.md

```diff
@@ -45,7 +45,7 @@ The command removes default projen task, makes projen remove its authority from
 ### Install
 Simply install dependencies:
 ```sh
-npm install
+npm ci
```

Comment:
@quesabe but this does clean install, with no cache. Why are we switching to such installation mode?

Reply:
@gvidon 
I don't see any mentions of no cache usage in [clean install](https://docs.npmjs.com/cli/v9/commands/npm-ci) - seems like it should use the cache (if it exists). The main advantage of clean install is that it never writes `package(-lock)?.json` and checks for inconsistent lock file (while install would rewrite). I just thought that it would be good to be informed of such cases. Overall I see that it's not that important in development and install can easily be used - inconsistent lock file would anyway be caught in CI.

If you see `install` as a better option for development, let me know and I'll switch it back.